CMakeLists.txt

```diff
@@ -380,21 +380,20 @@ if(NOT SKIP_INSTALL_ALL)
 		PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 endif()
 
-find_package(Doxygen)
-if(DOXYGEN_FOUND)
-	option(WITH_DOC "Generate documentation with Doxygen" ON)
-
-	# It is not an error when 'dot' is not found, just switching off the Doxygen's HAVE_DOT option
-	find_package_handle_standard_args (Dot REQUIRED_VARS DOXYGEN_DOT_EXECUTABLE)
-
-	include(cmake/CheckCaseSensitiveFileSystem.cmake)
-	if (HAVE_CASE_SENSITIVE_FILESYSTEM)
-		set(CMAKE_CASE_SENSITIVE_FILESYSTEM "YES")
-	else()
-		set(CMAKE_CASE_SENSITIVE_FILESYSTEM "NO")
-	endif()
+option(WITH_DOC "Generate documentation with Doxygen" OFF)
+if(WITH_DOC)
+	find_package(Doxygen)
```

Comment:
Alternatively, you could add the REQUIRED option to find_package(). If doxygen isn't found, it will fail at this point, and print an error message. And it won't be necessary to interrogate DOXYGEN_FOUND and to print a FATAL_ERROR message.
This would simplify the code, functionality remains the same.

Reply:
Done